composer/src/lib.rs

```diff
@@ -708,7 +708,7 @@ impl Drop for ComposeTest {
                     .output()
                     .unwrap();
                 std::process::Command::new("docker")
-                    .args(&["rm", c])
+                    .args(&["rm", c, "-v"])
```

Comment:
Shouldn't the `-v` come before the list of containers?

Reply:
oddly seemed to work anyway, but yes it should come before so I've made the change :+1: